4337/src/utils/userOp.ts

```diff
@@ -156,32 +166,28 @@ export const buildUserOperationFromSafeUserOperation = ({
 }
 
 export const getRequiredGas = (userOp: UserOperation): string => {
-  let multiplier = 3
-  if (userOp.paymasterAndData === "0x") {
-    multiplier = 1
+  let multiplier = 3n
+  if (userOp.paymasterAndData === '0x') {
+    multiplier = 1n
   }
 
-  return BigNumber.from(userOp.callGasLimit)
-    .add(BigNumber.from(userOp.verificationGasLimit).mul(multiplier))
-    .add(userOp.preVerificationGas)
-    .toString()
+  return (BigInt(userOp.callGasLimit) + BigInt(userOp.verificationGasLimit) * multiplier + BigInt(userOp.preVerificationGas)).toString()
 }
 
 export const getRequiredPrefund = (userOp: UserOperation): string => {
-  console.log({
-    requiredGas: getRequiredGas(userOp),
-    requiredPrefund: BigNumber.from(getRequiredGas(userOp)).mul(BigNumber.from(userOp.maxFeePerGas)).toString(),
-  })
+  const requiredGas = getRequiredGas(userOp)
+  const requiredPrefund = (BigInt(requiredGas) * BigInt(userOp.maxFeePerGas)).toString()
+  console.log({ requiredGas, requiredPrefund })
 
-  return BigNumber.from(getRequiredGas(userOp)).mul(BigNumber.from(userOp.maxFeePerGas)).toString()
+  return requiredPrefund
 }
 
 export const calculateIntermediateTxHash = (callData: string, nonce: BigNumberish, entryPoint: string, chainId: BigNumberish): string => {
```

Comment:
This can also be removed, I think. We no longer have intermediate tx hashes because the validated transactions are tracked in the entry point now.

Reply:
It is indeed not used anywhere else.